_kbarticles/2022-12-08-common-kibana-searches-on-clouddotgov.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Understanding Kibana and how to visualize your application logs"
+date: December 8, 2022
+excerpt: This article briefly explains what Kibana does, what types of logs are available in Kibana, and how to create log visualizations in Kibana.
+
+---
+
+## What does Kibana do?
+
+Kibana is a user interface that lets you search and visualize your application logs. Kibana has a [user guide](https://www.elastic.co/guide/en/kibana/current/index.html) that explains more about how to use it and to create custom visualizations.
+
+## What are some types of logs?
+
+Cloud Foundry assigns a type to each log message depending on its origin. Application logs are assigned the APP log type.  HTTP requests being routed to an app will produce the RTR log type. The various types of logs are listed in the documentation [here](https://docs.cloudfoundry.org/devguide/deploy-apps/streaming-logs.html#format).
+
+Cloudfoundry logs are often translated into field names in Elasticsearch/Kibana using the log type as a prefix. For example, APP logs in Kibana include fields like `app.name` for the application name.
+
+## How to chart the information:
+
+Spikes in traffic can be visualized by using the chart provided in Kibana. To find the specific information you are requesting you need to use the Discover tab to search RTR logs, and then create the chart based on the saved search. Use a count aggregation as the Y-Axis and a date histogram aggregation for the X-Axis.
+
+## Discover Search
+![Discover Search](https://user-images.githubusercontent.com/104385372/207403558-68266274-af72-43e1-b869-c9ad1805adca.png)
+
+## Add Timestamp Visualization
+![Add Timestamp Visualization](https://user-images.githubusercontent.com/104385372/207403643-d2aa8bb8-702b-4aee-a028-baa7caf25fee.png)
+
+## Add Date Histogram Aggregation for X-Axis
+![Add Date Histogram Aggregation for X-Axis](https://user-images.githubusercontent.com/104385372/207403718-60e70970-7dd4-452f-9d57-8c2230a5c450.png)
```

Comment:
```suggestion
```

This screenshot is misleading because it shows a line chart when it fact a histogram is produced by default. I think it might be easier to just exclude it.

Reply:
I agree